test/integration/acceptance/set_account_detail_test.cpp

```diff
@@ -147,3 +167,69 @@ TEST_F(SetAccountDetail, WithGrantablePerm) {
         ASSERT_EQ(block->transactions().size(), 1);
       });
 }
+
+/**
+ * C276
+ * @given a user with required permission
+ * @when execute tx with SetAccountDetail command with max key
+ * @then there is no tx in proposal
```

Comment:
I don't like that we say "no tx in proposal" while actually checking the block. We not have the `checkVerifiedProposal(..)`, so maybe use it or write "no tx in block"?